src/libs/actions/Policy.ts

```diff
@@ -960,6 +974,101 @@ function removeMembers(accountIDs: number[], policyID: string) {
     API.write(WRITE_COMMANDS.DELETE_MEMBERS_FROM_WORKSPACE, params, {optimisticData, successData, failureData});
 }
 
+/** Leave a workspace */
```

Comment:
We can remove this. name implies the reason
```suggestion
```

Reply:
Done - 7321ad2.